src/pages/console/data/content-management.mdx

```diff
@@ -68,12 +68,12 @@ Alternatively, you can save save *All About Dogs* before linking to an Author, a
 
 ## Limitations
 
-### Conflict Resolution
-Amplify Studio's data manager requires your app to have Conflict Resolution enabled. 
+### Conflict Resolution and DataStore
+Studio's Data Manager supports Amplify GraphQL APIs with and without DataStore. However, GraphQL APIs without DataStore have several minor limitations:
+- Data cannot be sorted by column
+- The search bar is disabled
```

Comment:
Should not being able to open an app in multiple tabs be included as a limitation for DataStore apps?

Reply:
This section exists, but in a different spot - moved it to be right after the Conflict Resolution setting.